app/services/legal_framework/organisation_types/all.rb

```diff
@@ -0,0 +1,22 @@
+module LegalFramework
+  module OrganisationTypes
+    class All < LegalFramework::BaseApiCall
+      class OrganisationTypeStruct
+        attr_reader :ccms_code, :description
+
+        def initialize(ot_hash)
```

Comment:
naming: could we make the param `ot_hash` more descriptive?

Reply:
Have renamed ot_hash to organisation_type_hash.